packages/nimble-components/src/toggle-button/template.ts

```diff
@@ -14,7 +15,7 @@ ButtonOptions
     <div
         role="button"
         part="control"
-        tabindex="${x => (x.disabled ? null : 0)}"
+        tabindex="${x => (x.disabled ? null : x.tabIndex ?? 0)}"
```

Comment:
A couple lines below we set the disabled property of this `div` but the `div` element does not have a disabled property

Reply:
Can you also fix the whitespace for the comments below? Guessing someone just let prettier win